CHANGELOG.md

```diff
@@ -5,42 +5,45 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased][unreleased]
```

Comment:
In my PR, I had this as `## [Unreleased]` (with updated footnote link to match).  I think this matches better with the other `##` headers.  Thoughts?